core/source/Console.mint

```diff
@@ -0,0 +1,282 @@
+store Console.Counter {
+  state count : Number = 0
+
+  fun increment {
+    next { count: count + 1 }
+  }
+
+  fun clear {
+    next { count: 0 }
+  }
+}
+
+/* Module to work with the [Console API](https://developer.mozilla.org/en-US/docs/Web/API/Console_API). */
+module Console {
+  /*
+  If the assertion is false, the message is written to the console. Supports string substitution.
+
+  [assert()](https://developer.mozilla.org/en-US/docs/Web/API/console/assert)
+  */
+  fun assert (assertion : Bool, value : a, values : Array(b) = []) : Tuple(Bool, a, Array(b)) {
+    `console.assert(#{assertion}, #{value}, ...#{values})`
+
+    {assertion, value, values}
+  }
+
+  /*
+  Clears the console.
+
+  [clear()](https://developer.mozilla.org/en-US/docs/Web/API/console/clear)
+  */
+  fun clear : Void {
+    `console.clear()`
+  }
+
+  /*
+  Logs the number of times that this particular call to count() has been called.
+  Returns the label passed along with the current count.
+
+  [count()](https://developer.mozilla.org/en-US/docs/Web/API/console/count)
+  */
+  fun count (label : String = "Default") : Tuple(String, Number) {
+    `console.count(#{label})`
+    Console.Counter.increment()
```

Comment:
The counter is global, not based on the label, I think that wasn't the intention :slightly_smiling_face: 

You need to change the state in the store to accommodate this:
```
state counts : Map(String, Number) = Map.empty()
```